lib/hashie/extensions/persistable.rb

```diff
@@ -0,0 +1,38 @@
+  module Hashie
+    module Extensions
+      module Persistable
+        CANNOT_INCLUDE = 'Peristable can only be used on classes with Hashie::Extensions::HashInitializer'
```

Comment:
We don't do this around Hashie as far as I can see. Maybe just specialize an exception that inherits from StandardError?